20220708.md

```diff
@@ -3,37 +3,37 @@ Repository activities from 2022-06-24 to 2022-07-08
 
 # Repository roundup
 
-A lot of repositories (several tens) has seen the creation of a new `upgrade-4.4.0` branch. This has often been used to backport fixes from the development branch to a given SailfishOS version, precluding (or not) a hot fix release like the 4.4.0.64 that happened a bit after 4.4.0.58 was out. These new branches didn't receive any backport commit yet, though, meaning that they may be used for something else than a hot fix release. Let's wait and see.
```

Comment:
I forgot to delete this intro when I copy-pasted from the previous digest. But actually, it not that bad since many repos still received branch updates. But I would change it a bit to something like :
In the continuation of last weeks, a lot of repositories (several tens) have seen the creation of a new `upgrade-4.4.0` branch. This has often been used to backport fixes from the development branch to a given Sailfish OS version, precluding (or not) a hot fix release like the 4.4.0.64 that happened a bit after 4.4.0.58 was out. A majority of these new branches have received a simple submodule URL change, but some has actually seen a backport commit like `connman` repo for instance.

Reply:
Since the text still fitted I thought it was intentional! But your changes are nice, and I've added them to the commit. Thank you.